libs/editor/build.gradle

```diff
@@ -34,6 +34,13 @@ android {
         targetSdkVersion rootProject.targetSdkVersion
     }
 
+    buildTypes {
+        debug {
+            minifyEnabled true
```

Comment:
Just and FYI and as per [the documentation](https://developer.android.com/build/shrink-code#enable) when `minifyEnabled` is set to true then this enables code shrinking, obfuscation, and optimization for only your project's release build type. Make sure to use a build variant with 'isDebuggable=false'.

PS: I am not sure if that's what you want and whether this will help the editor to load faster. 🤔

Reply:
Thanks Petros, this code was a hacky experiment to see if I could "trick" the debug variant into matching [the main app's release variant](https://github.com/wordpress-mobile/WordPress-Android/blob/26f817f6a2ab3891bc5108716b2e797bd4e9b133/WordPress/build.gradle#L262-L268C10). 😅 It didn't appear to have any impact on the editor's load time, and from your comments it does seem like it isn't a viable approach to continue experimenting with. Really appreciate your expertise here. 🙇‍♀️